src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,75 @@
+package core.basesyntax;
+
+import dao.FruitStoreDao;
+import dao.FruitStoreDaoImpl;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import model.StoreOperation;
+import service.CreatingFileService;
+import service.FileReaderService;
+import service.FileWriterService;
+import service.ReportService;
+import service.impl.CreatingFileServiceImpl;
+import service.impl.FileReaderServiceImpl;
+import service.impl.FileWriterServiceImpl;
+import service.impl.ReportServiceImpl;
+import strategy.StoreOperationStrategy;
+import strategy.StoreOperationStrategyImpl;
+import strategy.handler.BalanceOperationHandler;
+import strategy.handler.OperationHandler;
+import strategy.handler.PurchaseOperationHandler;
+import strategy.handler.ReturnOperationHandler;
+import strategy.handler.SupplyOperationHandler;
+
+public class Main {
+    public static void main(String[] args) {
+
+        // Creating new database file
+        String storeFilePath = "src/main/resources/friutshopdatabase.csv";
+        CreatingFileService creatingFileStore = new CreatingFileServiceImpl();
+        creatingFileStore.createFile(storeFilePath);
+
+        // Writing data to file
+        String data = "type,fruit,quantity\n"
+                + "b,banana,20\n"
+                + "b,apple,100\n"
+                + "s,banana,100\n"
+                + "p,banana,13\n"
+                + "r,apple,10\n"
+                + "p,apple,20\n"
+                + "p,banana,5\n"
+                + "s,banana,50";
+        FileWriterService fileWriter = new FileWriterServiceImpl();
+        fileWriter.writeDataToFile(storeFilePath, data);

```

Comment:
You do not need to create this file in main() - just put this file to src/main/resources and add a constant to this class with the path "src/main/resources/transactions.csv"